jflex/src/main/java/jflex/StatePairList.java

```diff
@@ -69,7 +69,6 @@ public void markAll(StatePairList[][] list, boolean[][] equiv) {
 
   private void increaseSize(int length) {
     length = Math.max(length + 1, 4 * p.length);
-    Out.debug("increasing length to " + length); // $NON-NLS-1$
```

Comment:
But debug cannot be static in a thread-safe world (or I'm missing something)

Reply:
That's true, this will only work for debugging the single-threaded case, but that's the only case you really want to debug the algorithmic part on. If we are getting bugs that are algorithm-dependent and only occur in the multi-threaded setting, we're screwed anyway, i.e. we need to make sure every generation thread works on its own private set of data structures (which this PR is doing, so that should be fine).

Reply:
Using `System.out` now.

Reply:
Arg! The regressions tests were relying on the output of stdout. So also reverted, and using instances of OUt.